dmoj/executors/RUST.py

```diff
@@ -12,55 +12,10 @@
 [dependencies]
 dmoj = "0.1"
 rand = "0.3"
-"""
-
-CARGO_LOCK = b"""\
-cargo-features = ["strip"]
-
-[root]
-name = "user_submission"
-version = "1.0.0"
-dependencies = [
- "dmoj 0.1.5 (registry+https://github.com/rust-lang/crates.io-index)",
- "rand 0.3.15 (registry+https://github.com/rust-lang/crates.io-index)",
-]
 
 [profile.release]
```

Comment:
Can you confirm that you see binaries getting stripped after this change? I see you also removed `cargo-features = ["strip"]` above.

Reply:
Sure, let me go double check.

Reply:
Here's the relevant manifest reference entry https://doc.rust-lang.org/cargo/reference/profiles.html#strip

Reply:
I get:

```
~/Projects/dmoj/judge/judge-server$ file user_submission 
user_submission: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=60849cf0f042bba9a714ff8060249b92104a97c4, for GNU/Linux 3.2.0, stripped
```

Reply:
I added the relevant `strip` section to the TOML, and it seems binaries are indeed stripped, so we should be all good.